### PR TITLE
Add keywords to NTS PDF download

### PR DESCRIPTION
**Pull request by lennym**

### CODE REVIEW

**Comment by lennym** [comment]
There's a failing test for this at https://github.com/UKHomeOffice/asl-autoproject/pull/47

File: pages/project-version/pdf/views/nts/v0.jsx [hunk 1]
```diff
@@ -37,6 +37,15 @@ export default function SchemaV0() {
         />
       </div>
 
+      <div className="q-and-a">
+        <h3>Key words</h3>
+        <p>{
+          (version.keywords || []).length
+            ? (version.keywords || []).join(', ')
```
Comment (wheelsandcogs) [inline pages/project-version/pdf/views/nts/v0.jsx:44]:
```suggestion
            ? version.keywords.join(', ')
```

File: pages/project-version/pdf/views/nts/v1.jsx [hunk 2]
```diff
@@ -74,7 +74,11 @@ export default function SchemaV1() {
 
       <div className="q-and-a">
         <h3>Key words</h3>
-        <p>No answer provided.</p>
+        <p>{
+          (version.keywords || []).length
+            ? (version.keywords || []).join(', ')
```
Comment (wheelsandcogs) [inline pages/project-version/pdf/views/nts/v1.jsx:79]:
```suggestion
            ? version.keywords.join(', ')
```
same as above, no point checking twice?

Comment (lennym) [reply pages/project-version/pdf/views/nts/v1.jsx:79]: Valid point.

**Pull request merged**